test/js/pageobjects/Welcome.js

```diff
@@ -1,29 +1,35 @@
 import Base from './BasePage.js'
-import {locale, verifyElementCopy} from '../utils/mochaw'
+import { locale, verifyElementCopy } from '../utils/mochaw'
+import { testFocusManagement, elementCanReceiveFocus } from '../utils/accessibility'
 
 class Welcome extends Base{
     getPrimaryBtn() { return this.$('.onfido-sdk-ui-Button-button'); }
     get welcomeTitle() { return this.$('.onfido-sdk-ui-PageTitle-titleSpan')}
     get welcomeSubtitle() { return this.$('.onfido-sdk-ui-Welcome-text')}
     get footer() { return this.$('.onfido-sdk-ui-Theme-footer')}
     get primaryBtn() { return this.$('.onfido-sdk-ui-Button-button')}
-    
+
     copy(lang) {return locale(lang) }
-    
+
     verifyTitle (copy) {
         const welcomeScreenStrings = copy.welcome
         verifyElementCopy(this.welcomeTitle, welcomeScreenStrings.title)
         return this.welcomeTitle
     }
 
+    async verifyFocusManagement() {
```

Comment:
Why do `verifyFocusManagement` and `verifyIdentityButton` need to be asynchronous?

Reply:
they all should be async actually, since all logic with the webdriverjs is async

Reply:
Yes, otherwise you can lose the reference to the elements since webdriverjs works asynchronously